end_to_end_tests/scenario_prioritization.py

```diff
@@ -114,9 +114,8 @@ def make_chat_request(client: HttpSession, low_priority: bool, max_tokens: int =
     try:
         headers = {
             "ocp-apim-subscription-key": apim_subscription_one_key,
+            "x-priority": "low" if low_priority else "high",
```

Comment:
Reverted! I re-looked at the code and saw that it would output either `"low"` or `"high"` for `x-gw-priority` now so no need to explicitly pass `"high"` in for the header to get the queries to work (my initial thought).